Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/leader.yml

```
@@ -164,6 +164,8 @@
       entries:
       - id: CMMagazinePistolMK80
         points: 3
+      - id: RMCMagazinePistolSU6
+        points: 3
```

On latest 13 this seems to be 6
```suggestion
        points: 6
```